src/GSoft.Extensions.MediatR.Analyzers/AnalyzerReleases.Shipped.md

```diff
@@ -0,0 +1,18 @@
+## Release 0.1.0
+
+### New Rules
+
+Rule ID | Category | Severity | Notes
+--------|----------|----------|-------
+GMDTR01 | Naming | Warning | NamingConventionAnalyzer, [Documentation](https://github.com/gsoft-inc/gsoft-extensions-mediatr)
+GMDTR02 | Naming | Warning | NamingConventionAnalyzer, [Documentation](https://github.com/gsoft-inc/gsoft-extensions-mediatr)
```

Comment:
Should we have a specific documentation file per rule? I'm fairly certain that is how it works for recent warnings that I needed to see the help documentation for.

Reply:
It's true that popular Roslyn analyers usually have a markdown documentation page per rule ([StyleCop](https://github.com/DotNetAnalyzers/StyleCopAnalyzers/tree/master/documentation), [Meziantou.Analyzers](https://github.com/meziantou/Meziantou.Analyzer/tree/main/docs/Rules)...). I've chosen not to do this here for the following reasons:
* Roslyn analyzers are fairly new to us, people that can develop and maintaining these can be counted on one hand
* I'm afraid that this documentation becomes out-of-date (even in its current form in the README.md)
* These rules are quire simple, like "you should provide a cancellation token", or "use the generic version instead".